homeassistant/components/agent_dvr/camera.py

```diff
@@ -102,11 +102,14 @@ async def async_update(self):
             if self.device.client.is_available and not self._removed:
                 _LOGGER.error("%s lost", self.name)
                 self._removed = True
-        self._attr_available = self.device.client.is_available
         self._attr_icon = "mdi:camcorder-off"
         if self.is_on:
             self._attr_icon = "mdi:camcorder"
-        self._attr_extra_state_attributes = {
+
+    @property
+    def extra_state_attributes(self):
```

Comment:
Why doesn't it work to use `_attr_extra_state_attributes`?

Reply:
Ok, I see what's going on now. The entity stopped polling so `async_update` wasn't called and the `_attr_` attributes were never updated. So the only needed fix is to implement `should_poll` property.